packages/client/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # @lens-protocol/client
 
+## 1.3.0-next.0
+
+### Minor Changes
+
+- c2a91ef4: **Added** `Profile.invitedBy` field
+- 4c4505d2: **Added** support for Profile Guardian
+
+### Patch Changes
+
+- 0f75f79d: **Added** experimental `useCurrentSession` hook
+  **Fixed** issue with `Profile` entity being leaked by the `useWalletLogin` hook
+  **Fixed** bug preventing logout on expired credentials detected at startup type
```

Comment:
what do we do with these entries? they are not related to client SDK, might be the result of changes in underlying packages, but I don't think so in this case

Reply:
Good catch, manually fixed.